src/keycloak/Keycloak.Seeding/BusinessLogic/SeedDataHandler.cs

```diff
@@ -125,6 +125,16 @@ public IReadOnlyDictionary<string, string> ClientsDictionary
         get => _keycloakRealm?.ClientScopeMappings?.FilterNotNullValues().Select(x => (x.Key, x.Value)) ?? Enumerable.Empty<(string, IEnumerable<ClientScopeMappingModel>)>();
     }
 
+    public IEnumerable<(string ProviderType, ComponentModel ComponentModel)> RealmComponents
+    {
+        get => _keycloakRealm?.Components?.FilterNotNullValues().SelectMany(x => x.Value.Select(v => (x.Key, v))) ?? Enumerable.Empty<(string, ComponentModel)>();
+    }
+
+    public IEnumerable<(string Locale, IEnumerable<KeyValuePair<string, string>> Translations)> RealmLocalizations
+    {
+        get => _keycloakRealm?.LocalizationTexts?.FilterNotNullValues().Select(x => (x.Key, x.Value.Select(kvp => kvp))) ?? Enumerable.Empty<(string, IEnumerable<KeyValuePair<string, string>>)>();
```

Comment:
not 100% sure, but as IDictionary<K,V> implements IEnumerable<KeyValuePair<K,V>> I think a cast would also do:
```suggestion
        get => _keycloakRealm?.LocalizationTexts?.FilterNotNullValues().Select(x => (x.Key, x.Value as IEnumerable<KeyValuePair<string, string>>)) ?? Enumerable.Empty<(string, IEnumerable<KeyValuePair<string, string>>)>();
```

Reply:
adjusted